tests/test_spans_api.py

```diff
@@ -186,6 +186,27 @@ def test_count_for_project(self) -> None:
         assert response.status_code == 200, response.data
         assert data["data"][0]["aggregate"] > 10, data
 
+    def test_alias(self) -> None:
```

Comment:
This is probably not the the right way to test the change, but just wanted to validate that it looked the way I expected and I thought it might be helpful in reviewing